packages/components/filters/src/filter-menu/trigger-button/trigger-button.styles.ts

```diff
@@ -0,0 +1,109 @@
+import { css } from '@emotion/react';
+import { designTokens } from '@commercetools-uikit/design-system';
+
+export const triggerWrapper = css`
+  color: ${designTokens.colorNeutral40};
+  background-color: ${designTokens.colorSurface};
+  font-size: ${designTokens.fontSize20};
+  font-weight: ${designTokens.fontWeight500};
+  position: relative;
+  display: inline-flex;
+  align-items: center;
+  gap: ${designTokens.spacing20};
+  padding: 3px ${designTokens.spacing25};
+  border-radius: ${designTokens.borderRadius20};
+  box-shadow: 0 0 0 ${designTokens.borderWidth1} ${designTokens.colorNeutral85};
+  cursor: pointer;
+  height: ${designTokens.spacing50};
+  max-width: 100%;
+  align-self: flex-start;
+  &:hover {
+    background-color: ${designTokens.colorPrimary98};
+    .ui-kit-filter-trigger-badge-container {
```

Comment:
What is the plan for the `.ui-kit-filter-trigger-badge-container` selectors? Are they temporary? I feel there is no need for hardcoding classNames, the states can most likely be part of the `badgeContainer` style definition with a bit of tinkering.

Reply:
it's so that the parent container can change the color of the gradient in the child